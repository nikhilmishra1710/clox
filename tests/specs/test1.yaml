id: CLOX-001

# skip: true
args: 
  - arg1
  - arg2

input:
  type: 
    inline
  inline:
    10 13
  

expected_stdout:
  type:
    file
  file:
    tests/expected/test1.out.expected
  inline:
    - The sum is 23
    - The product is 130
  
expected_stderr:
  type:
    file
  file:
    tests/expected/test1.err.expected

expected_ec: 65

hooks:
  pre:
    type:
      file
    file:
      - tests/inputs/test1.pre.hook
  post:
    type:
      inline
    inline:
      - echo "This is a post-hook"

timeout: 10