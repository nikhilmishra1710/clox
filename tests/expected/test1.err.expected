This is a sample error message
